instrumentation/jdbi3/src/main/java/brave/jdbi3/Jdbi3BravePlugin.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.jdbi3;
+
+import java.time.Instant;
+
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.spi.JdbiPlugin;
+import org.jdbi.v3.core.statement.SqlStatements;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.jdbi.v3.core.statement.StatementContextListener;
+
+import brave.Span;
+import brave.Tracing;
+
+/**
+ * Instrumentation for JDBI 3.
+ 
+ * JDBI plugin that sends a trace span for every SQL command executed by Jdbi.
+ * Install by calling {@code jdbi.installPlugin(new Jdbi3BravePlugin(tracing));}
+ */
+public class Jdbi3BravePlugin extends JdbiPlugin.Singleton {
+    private final Tracing tracing;
+    private final boolean includeBindVariables;
+
+    public Jdbi3BravePlugin(Tracing tracing) {
+            this(tracing, false);
+    }
+
+    public Jdbi3BravePlugin(Tracing tracing, boolean includeBindVariables) {
+            this.tracing = tracing;
+            this.includeBindVariables = includeBindVariables;
+    }
+
+    @Override
+    public void customizeJdbi(Jdbi jdbi) {
+        if (tracing == null) {
+            return;
+        }
+        jdbi.getConfig(SqlStatements.class)
+            .addContextListener(new StatementContextListener() {
+                @Override
+                public void contextCreated(final StatementContext ctx) {
+                    String spanName = "jdbi." + ctx.describeJdbiStatementType();
```

Comment:
hmm try to match the data policy with mysql. I don't think we want to put jdbi in the span name.. https://github.com/openzipkin/brave/blob/master/instrumentation/mysql8/src/main/java/brave/mysql8/TracingQueryInterceptor.java